apps/omg_eth/lib/omg_eth/ethereum_height_monitor.ex

```
@@ -90,21 +90,9 @@ defmodule OMG.Eth.EthereumHeightMonitor do
     {:noreply, state}
   end
 
-  # we need to figure out where :sslsocket is getting closed
-  # because it randomly kills this process and block getter
-  # https://github.com/omisego/elixir-omg/issues/1529
-  # https://sentry.io/organizations/omisego/issues/1582175010/?project=1477673
-  def handle_info({:ssl_closed, {:sslsocket, {:gen_tcp, port, :tls_connection, :undefined}, pids}}, state) do
-    port_info = Port.info(port)
-    _ = Logger.error("Getting ssl closed from port #{inspect(port_info)}")
-
-    _ =
-      Enum.map(pids, fn pid ->
-        process_info = Process.info(pid)
-        _ = Logger.error("Affected pids #{inspect(process_info)}")
-      end)
-
-    {:stop, :killed_by_ssl_closed, state}
+  def handle_info({:ssl_closed, _}, state) do
```

lol 😢
consider to still log the occurrence for us to understand the impact better though.
Also, are we confident we don't need to let the process crash in such error happens?

---

we'll see in `development` :)